### DISCUSSION TITLE
How to get the tracebook of an error in docker servive logs?

### DISCUSSION THREAD

**Discussion by gregsifr** [tiangolo/uvicorn-gunicorn-fastapi-docker#309] [Questions] [answered]
If there is an error in the fastapi application code there is no traceback which shows what the error was or where it occurred?

How to enable the traceback?

I am using the FastAPI template from the FastAPI documentation.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

That also means that the configuration should be a lot simpler and the result much more predictable as it's just Uvicorn, not Gunicorn on top. :nerd_face:
